src/NuGet.Core/NuGet.Protocol.Core.v3/LocalRepositories/LocalV2FindPackageByIdResource.cs

```diff
@@ -106,7 +106,7 @@ private List<CachedPackageInfo> GetPackageInfos(string id)
                         throw new FatalProtocolException(message, ex);
                     }
 
-                    if (string.Equals(reader.GetId(), id, StringComparison.Ordinal))
+                    if (string.Equals(reader.GetId(), id, StringComparison.OrdinalIgnoreCase))
```

Comment:
Both were introduced in .NET 2. After a bit of reading, I was not able to see a reason to use one over the other.